lib/messageFormat/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ *  @fileOverview
+ *  messageFormat provides utility functions to which use the INTL API which provides 
+ *  language sensitive string comparison, number formatting, and date and time formatting.
+ * 
+ *  messageFormat can have functions which are used in multiple places
+ *
+ *  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl
+ */
+
+
+/**
+ * Generates the country name associated with the given locale
+ * @param {string} locale The locale code (e.g., 'en-US')
```

Comment:
Do we supporting en_US and en-US format? Update the example given in the param document.